hid/joystick_windows.h

```diff
@@ -122,50 +127,50 @@ Joystick::read(Event &js)
                 int rThumbXState2 = Read().Gamepad.sThumbRX;
```

Comment:
Re point 2: I think it is actually correct -- the code is trying to work out which buttons' states have changed since the last poll -- but will add comments.

I agree with the other points and will also give this file a bit of a tidy-up.